src/s1reader/s1_reader.py

```diff
@@ -56,7 +56,12 @@ def parse_polynomial_element(elem, poly_name):
 
     half_c = 0.5 * isce3.core.speed_of_light
     r0 = half_c * float(elem.find('t0').text)
-    coeffs = [float(x) for x in elem.find(poly_name).text.split()]
+
+    if elem.find(poly_name) is None:  # some data with old IPF version (e.g. 2.36)
```

Comment:
We are usually leaving comments at the top of each line of code, to make sure to be in the limits of the pep8 convention.

Reply:
Thanks for our suggestion. I've checked the style guide (https://peps.python.org/pep-0008/#inline-comments), and concluded that the inline comments would be helpful to describe which line is for the old format (and which one is for the new format). I've made sure that it passed the codacy.